lib/Cache.js

```diff
@@ -126,13 +126,46 @@ function Cache () {
   this.get = function (type, key, options, callback) {
     var self = this
     var table = type + ':' + key
-    this.db.select(table, options, function (err, result) {
+    var query = this.decodeGeoservices(options)
+    this.db.select(table, query, function (err, result) {
       if (err) return callback(err)
       self.process(type, key, result, options, callback)
     })
   }
 
   /**
+   * Translates a geoservices query into sql parts
+   *
+   * @param {object} query - an object with geoservices strings
+   * @return {object} parsed sql parts
+   */
+  this.decodeGeoservices = function (query) {
+    if (query.orderByFields && query.orderByFields.length) {
+      query.order_by = this._convertOrderByFields(query.orderByFields)
+      delete query.orderByFields
```

Comment:
This is not as critical here but I think it's a bad practice to delete properties off of an object that is passed in as a parameter. We do a lot of that directly on the request and response objects and it can make debugging and refactoring harder since inputs are being changed by processing functions rather than those functions creating something new.